src/libZilliqa/Zilliqa.cpp

```diff
@@ -148,6 +148,43 @@ Zilliqa::Zilliqa(const PairOfKey& key, const Peer& peer, SyncType syncType,
     LOG_STATE("[IDENT] " << string(key.second).substr(0, 8));
   }
 
+  // Logging useful information for seed nodes
+  if (syncType == SyncType::NEW_LOOKUP_SYNC) {
```

Comment:
`NEW_LOOKUP_SYNC` check applies only to check the lookup node (a.k.a lookups or seeds) sync status. To identify the seed node we can check `ARCHIVAL_LOOKUP` flag if true.
if(LOOKUP_NODE_MODE) = LOOKUP NODE(can be both lookups and seeds)
if(LOOKUP_NODE_MODE && ARCHIVAL_LOOKUP) = seed node